Wire fuses grounding (#55)
LibreSolar/learn.libre.solar: docs/system/wires_fuses.md

<figcaption><b>Table 2.</b> MCCC deviation factors for different temperatures.</figcaption>

### Installation conditions

Enclosing the wires in duct, trays or conduit will decrease the current carrying capacity of the wire, since heat builds up in such enclosed system. This issue can be relaxed by providing proper ventilation or cooling system. In contrast, installing wires in open air allows for higher current carrying capacity of the wire. When installing in open air, UV and moisture resistance has to be considered for the insulation material.

The interactive graph given below, helps us to understand how parameters like specific resistance ($\rho$), length ($L$) and cross-sectional area of the conductor ($A$) affects efficiency and voltage drop across it.

<figure>
    <wire-efficiency/>
    <br/>
<center>
    <figcaption><b>Figure 1.</b> Efficiency variations of electrical wires <b>(interactive)</b>.</figcaption>
</center>
</figure>

The voltage drop across the wire has to be considered when the [charge controller](charge_controller) is configured to maintain the proper voltage on the battery to charge/discharge.

<figure>
    <wire-voltage-drop/>
    <br/>
<center>

>     <figcaption><b>Figure 2.</b> Voltage drop across 20m long wires with different cross section area.</figcaption>

Reviewer: ```suggestion
    <figcaption><b>Figure 2.</b> Voltage drop across cables with different cross section.</figcaption>
```

</center>
</figure>

### Example

> The distance between the panels and the charge-controller shall be $D=5m$, which means a total length (two directions) of $l_{wire} = 10m$. For a copper wire with a diameter of $d=1.5mm$, the total resistance of the wire (for an ambient temperature of 30°C is

Reviewer: Mhh, the units are still italic here...

|   35        		    |        0.94        |
|   40        		    |        0.87        |
|   45        		    |        0.79        |
|   55       		    |        0.61        |
|   65        		    |        0.35        |
<figcaption><b>Table 2.</b> MCCC deviation factors for different temperatures.</figcaption>

### Installation conditions

Enclosing the wires in duct, trays or conduit will decrease the current carrying capacity of the wire, since heat builds up in such enclosed system. This issue can be relaxed by providing proper ventilation or cooling system. In contrast, installing wires in open air allows for higher current carrying capacity of the wire. When installing in open air, UV and moisture resistance has to be considered for the insulation material.

The interactive graph given below, helps us to understand how parameters like specific resistance ($\rho$), length ($L$) and cross-sectional area of the conductor ($A$) affects efficiency and voltage drop across it.

<figure>
    <wire-efficiency/>
    <br/>
<center>
    <figcaption><b>Figure 1.</b> Efficiency variations of electrical wires <b>(interactive)</b>.</figcaption>
</center>
</figure>

> The voltage drop across the wire has to be considered when the [charge controller](charge_controller) is configured to maintain the proper voltage on the solar panels.

Reviewer: More relevant for the battery and loads. In case of the solar panel, the voltage is generated by the solar panel itself and the charge controller can't do much to control it (except for trying to get maximum power). 

A battery would never be connected to the charge controller via 20 m wire. So it would be good to keep also this graph interactive and make the length configurable.

<figure>
    <wire-voltage-drop/>
    <br/>
<center>

>     <figcaption><b>Figure 1.</b> Voltage drop across 20m long wires with different cross section area.</figcaption>

Reviewer: ```suggestion
    <figcaption><b>Figure 2.</b> Voltage drop across 20m long wires with different cross section area.</figcaption>
```

</center>
</figure>

### Example

> The distance between the panels and the charge-controller shell be $D=5m$, which means a total length (two directions) of $l_{wire} = 10m$. For a copperwire with a diameter of $d=1.5mm$, the total resistance of the wire (for an ambient temperature of $30^{\circ}C$) is

Reviewer: ```suggestion
The distance between the panels and the charge-controller shall be $D=5m$, which means a total length (two directions) of $l_{wire} = 10m$. For a copper wire with a diameter of $d=1.5mm$, the total resistance of the wire (for an ambient temperature of $30^{\circ}C$) is
```

$$R=\frac{0.0172 \cdot 10}{1.5} \Omega = 0.114 \Omega$$

If we assume a maximum current of $I_{max} = 10A$ like charge controller [MPPT 1210 HUS](https://libre.solar/hardware/mppt-1210-hus.html) can handle, the maximum power dissipation in the wire is

$$P_{wire} = I_{max}^2 * R = 100A^2 * 0.114 \Omega = 11.4W$$

In most cases, this approach is rather used to determine the minimal cross section for the wires. If a power loss of 2% of the peak power is tolerated and the maximum expected temperature in the wire is assumed to be $50^{\circ}C$, the cross section can be determined using the above formulas. Let the peak power be $P_{peak}=120W$. The specific resistance at $50^{\circ}C$ is $\rho_{50} = 0.0185\frac{\Omega mm^2}{m}$.

$$P_{wire} = 2\%\cdot P_{peak} = 2.4W$$
$$R_{wire} = \frac{P_{wire}}{I_{max}^2} = \frac{2.4W}{100A} = 0.024\Omega$$
$$A_{wire} = \frac{\rho \cdot l_{wire}}{R_{wire}} = 7.7mm^2$$

So for this example, a wire with a cross section of $A=7.7mm^2$ should suffice.

For short connections below 5 meters, you can use:

> ::: warning Rule of Thumb:

Reviewer: Maybe better an information (green) and not a warning?

 current / 3 = cable cross section area
:::

Table 2 lists common cross section for some situations. Note that the length is given as the total of both positive and negative lines.

|                           | $length=5m$        |  $length=10m$|
|:-------------------------:|:------------------:|:------------------:|
|Cross section in $[mm^2]$  | max. current $[A]$ |  max. current $[A]$|
|   0.75        		    |        2.3         |      1.1
|   1.5        		        |        4.5         |      2.3
|   10        		        |        30          |      15
|   35        		        |        105         |      53
|   50        		        |        150         |      75

> |   120        		        |        360         |      180

Reviewer: Can you use spaces for indentation? This looks chaotic.

Author: I used spaces, I guess there were indentations already in there. Will fix it

### Branch

In case of branches where multiple lines are drawn from the main wire as shown in the Fig.3, a fuse has to be placed on each individual branch lines. Failure in doing so might result in excessive current in individual lines of the branch beyond their limit even when total current in the main line is within the limit. i.e, Fuse $F$ might have higher current rating than fuse $F_1$. In this case, even though when fuse $F$ doesn't trip and current is within the safe limit of the main line, fuse $F_1$ might trip due to excessive current in branch 1 beyond the limit of branch 1 line. Absence of $F_1$ in this case leads to damage of the systems.

<fig-caption src="system/fuse_branches.svg" caption="Position of fuse in branches" num="3" />

## Grounding

> Ground defines a common voltage level as a reference point, to which all other voltages are measured. This can be done by connecting all negative **or** positive sides to the same line, which can be a rod driven into the earth or the chassis of a car. To prevent accidents by faulty wires or electrical machines, it is common to connect non-electrical but conductive parts (e.g. machine housing) to ground as well. In case of a fault and a short circuit, current flows through the housing and into the earth rather than the body and through the legs into earth.

Reviewer: ```suggestion
The ground defines a common voltage level as a reference point, to which all other voltages are measured. This can be done by connecting all negative **or** positive sides to the same line, which can be a rod driven into the earth or the chassis of a car. To prevent accidents by faulty wires or electrical machines, it is common to connect non-electrical but conductive parts (e.g. machine housing) to ground as well. In case of a fault and a short circuit, current flows through the housing and into the earth rather than the body and through the legs into earth.
```

### Positive vs. negative grounding

> Which side of the circuit should be connected depends on the parts used. Most common is negative grounding, so that all voltages in the circuit are positive. Some components or combinations require negative voltages, so a positive grounding is required. Another reason to use positive grounding is electro-chemical corrosion. Corrosive effects are mainly driven by potential differences between metals or minerals, mostly in the presence of moisture. For wires buried in the ground or circuits in moist conditions, it could be beneficial to "pull" the potentials down.

Reviewer: Add reference to Load Switch chapter and high-side vs. low-side switching.

### RCD in AC networks

> To protect from currents leaking into earth, a residual current device should be installed. It measures ingoing and outgoing currents in the phase and neutral lines and triggers whenever a difference is detecting. The difference means that there is another way, mostly earth, for the current to flow than the designed one, indicating a potential hazard. When including an inverter in the system, the RCD has to be place **after** the inverter along with aditional switches. Refer to [this manual](https://www.victronenergy.com/upload/documents/Wiring-Unlimited-EN.pdf) for further eplanations.

Reviewer: Please proof-read to detect typos before submitting the PR ;)

```suggestion
To protect from currents leaking into earth, a residual current device should be installed. It measures ingoing and outgoing currents in the phase and neutral lines and triggers whenever a difference is detected. The difference means that there is another way, mostly earth, for the current to flow than the designed one, indicating a potential hazard. When including an inverter in the system, the RCD has to be placed **after** the inverter along with additional switches. Refer to [this manual](https://www.victronenergy.com/upload/documents/Wiring-Unlimited-EN.pdf) for further explanations.
```
